src/dev-server/client/hmr-util.ts

```diff
@@ -54,20 +56,54 @@ export const updateCssUrlValue = (versionId: string, fileName: string, oldCss: s
   return newCss;
 };
 
-export const isLinkStylesheet = (elm: Element) =>
+/**
+ * Determine whether a given element is a `<link>` tag pointing to a stylesheet
+ *
+ * @param elm the element to check
+ * @returns whether or not the element is a link stylesheet
+ */
+export const isLinkStylesheet = (elm: Element): boolean =>
   elm.nodeName.toLowerCase() === 'link' &&
   (elm as HTMLLinkElement).href &&
   (elm as HTMLLinkElement).rel &&
```

Comment:
Couldn't these be `undefined`? Wouldn't that make it such that the return type of this function could be `boolean | undefined` as a result?

Reply:
Could add `!!` to both those checks (which would also evaluate `false` if `href` is an empty string)

Reply:
mm true, I'll throw a `!!` in there